AccountRecords.cls

```diff
@@ -0,0 +1,12 @@
+public class AccountRecords{
+	public static void getAccountRecords(){
+		List<Account> accountList = new List<Account>();
+		accountList = [SELECT Id,Name FROM Account WHERE CreatedDate = TODAY];
+		
+		for(Integer a = 0; a < accountList.size(); a++){

```

Comment:
Here instead of traditional for loop you can go for `SOQL for loop` which work better in some scenarios